starlette_exporter/middleware.py

```diff
@@ -100,6 +102,16 @@ def __init__(
         self.app_name = app_name
         self.prefix = prefix
         self.group_paths = group_paths
+
+        if group_unhandled_paths and filter_unhandled_paths:
+            filter_unhandled_paths = False
+            warnings.warn(
+                "filter_unhandled_paths was set to True but has been changed to False "
+                "because group_unhandled_paths is True and these seetings are mutually exclusive",
```

Comment:
```suggestion
                "filter_unhandled_paths was set to True but has been changed to False "
                "because group_unhandled_paths is True and these settings are mutually exclusive",
```

Reply:
Thank you for spotting this.